src/main/java/optimus/TaskList.java

```diff
@@ -0,0 +1,50 @@
+package optimus;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class TaskList {
+    private List<Task> record;
+
+    public TaskList() {
+        this.record = new ArrayList<>();
+    }
+
+    public TaskList(List<Task> tasks) {
+        this.record = tasks;
+    }
+
+    public void addTask(Task task) {
+        record.add(task);
+    }
+
+    public void deleteTask(int index) {
+        record.remove(index);
+    }
+
+    public Task getTask(int index) {
+        return record.get(index);
+    }
+
+    public int sizeOfRecord() {
```

Comment:
Should the method name be modified to sound like a getter (e.g. getSizeOfRecord())?